content/en/docs/instrumentation/go/getting-started.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Getting Started
 weight: 10
+spelling: cSpell:ignore fibonacci struct fscanf chan println strconv stdouttrace
```

Comment:
There is at least one place in this file where Fibonacci is misspelled -- it should be capitalized, e.g.:

> // Fibonacci returns the n-th fibonacci number.

should be

> // Fibonacci returns the n-th Fibonacci number.

Reply:
In that sense, probably `fibonacci` (all in lowercase) should be removed from the `cSpell:ignore` list.

Reply:
fixed